packages/request-node/src/server.ts

```diff
@@ -1,42 +1,29 @@
-#!/usr/bin/env node
```

Comment:
this file now only creates the server object, but doesn't start it. startNode is called from bin.ts